clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx

```diff
@@ -52,6 +53,13 @@ const ConfigWizardWalkthrough = () => {
                 }
               />
             ) : null}
+            {step === 2 ? (
+              <AddSystemForm
+                handleChangeStep={(organizationStep: number) =>
+                  handleChangeStep(organizationStep)
+                }
```

Comment:
I've confirmed that the below does work, since the params will just automatically get passed through

```suggestion
                handleChangeStep={handleChangeStep}
```